services/github/github-package-lock-json.service.js

```diff
@@ -0,0 +1,126 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const {
+  isPackageLockJsonWithDependencies,
+  getLockDependencyVersion,
+} = require('../package-lock-json-helpers')
+const { fetchJsonFromRepo } = require('./github-common-fetch')
+const { documentation } = require('./github-helpers')
+const { BaseJsonService } = require('..')
+
+const keywords = ['npm', 'node']
+
+const dependencyQueryParamSchema = Joi.object({
+  filename: Joi.string(),
+}).required()
+
+class GithubPackageLockJsonDependencyVersion extends BaseJsonService {
+  static get category() {
+    return 'platform-support'
```

Comment:
Think this probably be under the `dependency` category. I know several of the other similar badges (npm dep version, github dep version from package.json, etc.) are in the `platform-support` category.

However, the pipenv locked dep version badge is under the `dependency` category and FWIW that feels like the better category for this badge (and those other ones currently under `platform-support`). 

@paulmelnikow  any thoughts on this?

Reply:
Hmm, interesting question.

This one is a lot like the Pipenv badge so whatever we decide, these probably belong in the same place.

Maybe we ought to narrow "Platform & Version Support" to "Platform Support" and reserve it for things like the Node version / php version, and move the others into Dependencies.